src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -26,6 +26,9 @@
 from snowflake.snowpark.table import Table
 from snowflake.snowpark.types import StructType, VariantType
 
+if TYPE_CHECKING:
+    from .column import Column
```

Comment:
It looks like in this repo, we usually use `snowflake.snowpark.column.Column` for annotations. Should we keep consistent?

Reply:
Sorry, I'll keep this in mind from now on!